scripts/setup/install_aws_dependencies.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+#
+# MIT License
+#
+# Copyright (c) 2023 EASL and the vHive community
```

Comment:
No EASL toched this file. Either put your name or HySCALE. Also, bump the date.

Reply:
Copyright (c) 2024 HySCALE

Is the above ok?